dxp/how_clay_arrives_to_dxp.md

```diff
@@ -0,0 +1,223 @@
+
+## How is Clay imported into a module?
+
+Unlike the previous section we won't need to install Clay as a dependency in a module inside Liferay Portal. Thanks to our rich environment, we already have it included and we just need to enable it.
+
+To do so, we need to define the new import into our dedicated buildCSS process, we can do that copy the following code into the `build.gradle` file of our module.
+
+```js
+buildCSS {
+	imports = [
+		new File(npmInstall.nodeModulesDir, "@clayui/css/src/scss")
+	]
+}
+```
+
+[Real example of buildCSS](https://github.com/liferay/liferay-portal/blob/master/modules/apps/adaptive-media/adaptive-media-web/build.gradle)
+
+## How to import Clay into a custom-module?
+
+Before unreavel the final piece of this first part, we must understand what's a module and what's the difference with a theme.
+
+In Liferay we refer to a module when we talk about an indipendent bundle that can be install in DXP. It usually contains configuration, markup and style. We often call them Widgets, Portlets or Themes.
+
+So as you probably already know by this moment, there is not (technical) difference between a custom-module and a custom-theme, and as for the custom-theme we can install Clay using
+
+Node
+`npm install @clayui/css`
+
+Yarn
+`yarn add @clayui/css`
+
+[Read more at Clay's website](https://clayui.com/docs/get-started/how-to-use-clay.html)
+
+## How to use Clay once imported?
+
+And finally we arrive at the section where we explain what to do with Clay once imported. In order to have Clay available in our CSS we'll need to import it in our `main.scss`, `style.scss` or `your-custom-name-css-file.scss`.
+
+It's a really simple code line:
+
+```scss
+@import 'atlas-variables';
+```
+
+Except, _it's not_ 😁.
+
+---
+
+There are 4 different ways to import Clay CSS.
+
+```scss
+@import 'base';
+@import 'base-variables';
+@import 'atlas';
+@import 'atlas-variables';
+```
+
+And, of course, each one of them has its unique characteristics.
+
+| Import          | Description                                                 |
+| --------------- | ----------------------------------------------------------- |
+| base            | All the functionalities of Clay with Bootstrap style        |
+| base-variables  | The main functions and variables of Clay in Bootstrap style |
+| atlas           | All the functionalities of Clay with Lexicon style          |
+| atlas-variables | The main functions and variables of Clay in Lexicon style   |
+
+[Learn everything about Clay CSS](https://github.com/liferay/liferay-frontend-guidelines/blob/281b28e93db2ade5a18230be8ac3b618f28cddb0/css/clay-css-3-intro.md)
+
+---
+
+There is a fifth _unofficial_ line that allow us to include a specific component's CSS we don't usually use it but if you are building something special you can use:
```

Comment:
We should mention we need `@import 'base-variables';` or `@import 'atlas-variables'` before importing the component CSS.

Reply:
allow -> allows